src/Driver/FlatFileDriver.php

```diff
@@ -10,8 +10,14 @@
  */
 class FlatFileDriver implements \Bernard\Driver
 {
+    /**
+     * @var string
+     */
     private $baseDirectory;
 
+    /**
+     * @var integer
```

Comment:
For now, private properties have no docblock on purpose. This might change, but in the meantime they should go away for consistency.